images/ipi-deprovision/Dockerfile

```diff
@@ -5,18 +5,12 @@ ARG JQ_VERSION=1.6
 
```

Comment:
Why do we need to keep this image here, shouldn't we simply make this a BuildConfig instead?

Reply:
All imagestreamtags in the CI get automatically distributed to all build clusters already: https://github.com/openshift/release/blob/8568b711c7374cd765c75d3086c92140e21a49d8/clusters/app.ci/assets/dptp-controller-manager.yaml#L126

So if you just put it there, nothing more to do.

Reply:
Also, there are 2 repo files that this image is using. I am not sure if we can inject them in the buildConfig.

Reply:
Let's take this approach for now. I will open a card to completely remove that image and use a bc.

Reply:
> Also, there are 2 repo files that this image is using. I am not sure if we can inject them in the buildConfig.

Yes, you can use `echo` in a `docker run` directive, I would assume a heredoc works as well. This image has literally nothing to do with this repo so I would very much prefer if we didn't build it on every PR here.

Reply:
https://github.com/openshift/release/pull/17065